test/manual/s3_test.cc

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+
+#include <boost/test/unit_test.hpp>
+#include <seastar/core/thread.hh>
+#include <seastar/core/reactor.hh>
+#include <seastar/core/file.hh>
+#include <seastar/core/fstream.hh>
+#include <seastar/testing/test_case.hh>
+#include "utils/s3/client.hh"
+
+/*
+ * Tests below expect minio server to be running on localhost
+ * with the bucket named "testbucket" created with unrestricted
+ * anonymous read-write access
+ */
+
+static const ipv4_addr s3_server("127.0.0.1", 9000);
+
```

Comment:
Let's move it away from test/manual and run minio. Integration into test.py will be annoying, but maybe it's possible to use boost.test fixures to launch minio directly from the test.

Reply:
Sure